src/config.js

```diff
@@ -39,6 +40,7 @@ const config: Config = {
     'update_message_flags',
     'user_status',
   ],
+  kandraHostedDomains: ['zulip.com', 'zulipchat.com', 'chat.zulip.org'],
```

Comment:
Thinking about the name for this variable: I'd like it to be more generically about the relationship of this (build of the) app to some domains, rather than Kandra by name. In principle if someone else makes and distributes their own build with their own App ID, they'd want to edit this to refer to domains they control, even though these would still be the ones hosted by Kandra.

(Even if nobody ends up ever doing that, I think it keeps the logic around this variable clearer to think about if it's named so it works in that general case.)

Perhaps `appAssociatedDomains`? `appOwnDomains`?

Reply:
Sure, makes a lot of sense!